tests/generic_views/test_detail.py

```diff
@@ -137,6 +140,34 @@ def test_duplicated_context_object_name(self):
         self.assertNotIn('author', res.context)
         self.assertTemplateUsed(res, 'generic_views/author_detail.html')
 
+    def test_deferred_queryset_template_name(self):
+        class FormContext(SingleObjectTemplateResponseMixin):
```

Comment:
not sure this class is really needed -- I assume it should be covered by other tests?